### ISSUE TITLE
Create "About" and "Contact" pages

### ISSUE THREAD

**Issue by niknow** [h2g2m/h2g2m#9]
The items "About" and "Contact" from the top menu bar are broken.

**Comment by larsborn** [comment]
no, they are not broken because they just show an 404 (Not Found) ... which is right because we don't have About and Contact pages yet.

**Comment by niknow** [comment]
Ok, but they are missing and should be created. ;)
